cypress/e2e/racepage.cy.js

```diff
@@ -1,5 +1,42 @@
-describe('template spec', () => {
-  it('passes', () => {
-    cy.visit('https://example.cypress.io')
-  })
-})
\ No newline at end of file
+describe('Home/Instructional Page', () => {
+  beforeEach('', () => {
+    cy.intercept('https://v1.formula-1.api-sports.io/rankings/drivers?season=2021', {
+      fixture: 'allDrivers.json'
+    })
+      .intercept('https://v1.formula-1.api-sports.io/drivers?name=Lewis%20Hamilton', {
+        fixture: 'lewis.json'
+      })
+      .intercept('https://v1.formula-1.api-sports.io/drivers?name=Max%20Verstappen', {
+        fixture: 'max.json'
+      })
+      .intercept('https://v1.formula-1.api-sports.io/rankings/teams?season=2021', {
+        fixture: 'teams2021.json'
+      })
+      .visit('http://localhost:3000/')
+  });
+
+  it('starts the user on an instruction page', () => {
+    cy.get('.formula-fun').contains('Welcome to Formula Fun!')
+      .get('p').contains('Get ready to take on the racing world')
+      .get('.see-team').contains('Build My Team');
+  });
+
+  it('allows users to click around each link in the header', () => {
+    cy.get('nav > [href="/team"]').click().get('.default-driver').should('exist');
+    cy.get('[href="/drivers"]').click().get('.full-rankings').should('exist');
+  });
```

Comment:
Are these links taking the user to a new page or just displaying different information? If they are taking the user to a new page consider checking the URL path with `cy.url('url path it should be at')`

Reply:
Oooh I didn't know that you could check a URL in Cypress!!  Genius